README.md

```diff
@@ -58,8 +58,9 @@ items:
   spec:
     ptpEventConfig:
       enableEventPublisher: true
-      transportHost: "http://ptp-event-publisher-service.openshift-ptp.svc.cluster.local:9043"
       # For AMQP transport, transportHost: "amqp://amq-router.amq-router.svc.cluster.local"
+      transportHost: "http://ptp-event-publisher-service-NODE_NAME.openshift-ptp.svc.cluster.local:9043"
```

Comment:
ptpoperatorconfig is a cluster level config ASAIK, but this change indicates we will only support SNO?

Reply:
its  not  full node name , it is just the first string for dot delimited string